docs/spl-docs/time-modifiers.md

```diff
@@ -0,0 +1,177 @@
+| `@q`, `@qtr`, or `@quarter` | Specify a snap to the beginning of the most recent quarter: Jan 1, Apr 1, July 1, or Oct 1. |
+| `w0`, `w1`, `w2`, `w3`, `w4`, `w5`, `w6`, and `w7` | Specify "snap to" days of the week, with `w0` representing Sunday, `w1` for Monday, and so on. Snapping to a week using `@w` or `@week` defaults to snapping to Sunday, equivalent to `@w0`. Sunday can be referred to as either `w0` or `w7`. |
+
+### Specify earliest relative time offset and latest time in ad hoc searches
+
+Ad hoc searches that use the earliest time modifier with a relative time offset should also include `latest=now` in order to avoid time range inaccuracies. For instance, to capture all events from the last 15 seconds starting at 01:00:15, the following search returns all events that occur between the time of 01:00:00 and 01:00:15, as expected.
+
+```spl
+index=main earliest=-15s latest=now
+```
+
+Running the same search without including `latest=now` might produce unpredictable results.
+
+## Examples
+
+### Examples of Absolute time modifiers 
+
+1. **Using `earliest` for a Specific Start Date**
+   To search for events starting from January 1, 2023, you would use:
+   ```spl
+   ... | earliest=01/01/2023:00:00:00
+   ```
+   In this search, `latest` is not specified which means that it would default to the current time `now()`. This search retrieves all events from the beginning of January 1, 2023, onwards
+
+2. **Combining `earliest` and `latest` for a Specific Date Range**
+   To search for events during the first quarter of 2023, you would use:
+   ```spl
+   ... | earliest=01/01/2023:00:00:00 latest=03/31/2023:23:59:59
+   ```
+   This command captures all events from January 1, 2023, up to and including March 31, 2023, providing a complete view of the first quarter's data.
+
+### Examples of relative time modifiers
+
+For the following examples, the current time is Monday, 07 October 2024, 10:15:00 A.M.
+
+> **Note:** `24h` is usually, but not always, equivalent to `1d` due to Daylight Savings Time adjustments.
+
+| Time Modifier | Description | Resulting Time | Equivalent Modifiers |
+|---------------|-------------|----------------|----------------------|
+| `now` | Now, the current time | Monday, 07 October 2024, 10:15:00 A.M. | `now()` |
+| `-60m` | 60 minutes ago | Monday, 07 October 2024, 09:15:00 A.M. | `-60m@s` |
+| `-1h@h` | 1 hour ago, to the hour | Monday, 07 October 2024, 09:00:00 A.M. |  |
+| `-1d@d` | Yesterday | Sunday, 06 October 2024, 12:00:00 A.M. |  |
+| `-24h` | 24 hours ago (yesterday) | Sunday, 06 October 2024, 10:15:00 A.M. | `-24h@s` |
+| `-7d@d` | 7 days ago, 1 week ago today | Monday, 30 September 2024, 12:00:00 A.M. |  |
+| `-7d@m` | 7 days ago, snap to minute boundary | Monday, 30 September 2024, 10:15:00 A.M. |  |
+| `@w0` | Beginning of the current week | Sunday, 06 October 2024, 12:00:00 A.M. |  |
```

Comment:
nit: On line 141, for the Equivalent Modifiers column, I think you mentioned previously that `@week` and `@w7` are the same as `@w0`, so you could include those